src/asynchronous/server.rs

```diff
@@ -303,63 +304,53 @@ async fn spawn_connection_handler<S>(
     });
 }
 
+async fn do_handle_request(
+    fd: RawFd,
+    methods: Arc<HashMap<String, Box<dyn MethodHandler + Send + Sync>>>,
+    header: MessageHeader,
+    body: Vec<u8>,
```

Comment:
It seemed there's no needs to transform the body's ownership, thus is it better to use the reference instead of moving the ownership?

Reply:
ok